CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# ebx-structuredlogging-sdk Changelog
```

Comment:
We want to mention version 1.0.0 in here with a release date of Jan 1, 2014. Can be identical to https://github.com/ebx/ebx-unixtime-sdk/blob/dev/CHANGELOG.md